webpack/webpack.config.js

```diff
@@ -73,19 +75,26 @@ module.exports = {
           }
         }
       },
+      {
+        test: /\.tsx?$/,
+        exclude: /node_modules/,
+        use: "ts-loader"
+      },
       {
         test: /\.js$/,
         exclude: /node_modules/,
-        use: {
-          loader: "babel-loader",
-          options: {
-            presets: requireAll([
-              "babel-preset-es2015",
-              "babel-preset-stage-3",
-              "babel-preset-react"
-            ])
+        use: [
```

Comment:
The change here is unnecessay, It can be an object instead of an array of object. I don't think it has a different meaning, but it's still something that just could stay the same

Reply:
you are right, that should go away.